### DISCUSSION TITLE
Jet Fan Louver Vector Output

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13019] [Q&A]
I would like to ask how the louver vector option works as I keep getting a wrong flow direction. I am specifying this vector for this output vent to flow in this direction however it flows in the opposite 
UVW=-4.2,-12.8,0.0/ 
![image](https://github.com/firemodels/fds/assets/96638521/660c0818-793a-4189-8011-02ac70be5152)

**Comment by marcosvanella** [comment]
I would try running that vent alone to see what the flow looks like and use vectors for velocity to see the direction better.

________________________________
From: ShehabGamrah ***@***.***>
Sent: Saturday, June 1, 2024 08:35 PM
To: firemodels/fds ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [firemodels/fds] Jet Fan Louver Vector Output (Discussion #13019)


I would like to ask how the louver vector option works as I keep getting a wrong flow direction. I am specifying this vector for this output vent to flow in this direction however it flows in the opposite
UVW=-4.2,-12.8,0.0/
image.png (view on web)<https://github.com/firemodels/fds/assets/96638521/660c0818-793a-4189-8011-02ac70be5152>

**Comment by Er9y714** [comment]
Are you running on the last FDS version 6.9.1?
can you share the input lines of that fan OBST and VENT?
